src/lib.rs

```diff
@@ -1,342 +1,3 @@
-//! lockc is open source software for providing MAC (Mandatory Access Control)
-//! type of security audit for container workloads.
-//!
-//! The main technology behind lockc is [eBPF](https://ebpf.io/) - to be more
-//! precise, its ability to attach to [LSM hooks](https://www.kernel.org/doc/html/latest/bpf/bpf_lsm.html)
-//!
-//! Please note that currently lockc is an experimental project, not meant for
-//! production environments. Currently we don't publish any official binaries or
-//! packages to use, except of a Rust crate. Currently the most convenient way
-//! to use it is to use the source code and follow the guide.
-//!
-//! If you need help or want to talk with contributors, please come chat with
-//! us on `#lockc` channel on the [Rust Cloud Native Discord server](https://discord.gg/799cmsYB4q).
```

Comment:
I would keep the old comment until this moment, as a short introduction in https://docs.rs/lockc/0.1.0/lockc/ and maybe add something like this at the beginning:

```
//! This is an auto-generated code documentation. For more detailed documentation
//! (with all information about usage, deployment and architecture) please check
//! out [the book](https://rancher-sandbox.github.io/lockc/).
```

Reply:
done